service_storagefacility/src/test/scala/no/uio/musit/microservice/storagefacility/service/StorageNodeServiceSpec.scala

```diff
@@ -438,7 +438,33 @@ class StorageNodeServiceSpec extends MusitSpecWithAppPerSuite with NodeGenerator
     val oldDataRes = service.getRoomById(mid, inserted.id.get).futureValue
     oldDataRes.get.get.securityAssessment.waterDamage mustBe Some(false)
   }
+    "get currentLocation on a object" in {
```

Comment:
Bad formatting / indentation.

It is a good habit to get used to doing `sbt -mem 2048 clean test it:test` on the command line before you commit and push your changes to git. At least as the last thing you do before creating a PR. That way, if you have code that isn't formatted correctly, it will be handled by the automatic code formatting tool.